### DISCUSSION TITLE
[ 2020 - Year in Review ] Grateful for everything ❤️

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [vinitshahdeo/vinitshahdeo#1] [Show and tell]
# 👋  Hello!

2020 : Thanks for the experiences, good or bad!
**2021 : Welcome, here I go!**

This year wasn't like any year before, but we learnt a lot about how we can work together in new ways to make things work. This time last year, concepts such as _"lockdown", "mask mandates"_ and _"social distancing"_ were unknown to most of us. Today they are part of our everyday language as the COVID-19 pandemic continues to impact all aspects of our lives.

2020, what a year this was! Despite all the problems we faced, **I got a chance to spend time with my family after 8 years.** ...and my elder sister [got married](https://github.com/vinitshahdeo/Wedding-Invitation) this year! I'm grateful for everything:) ❤️ 

### Here's a recap of my 2020:

- Completed [2 years at Postman](https://www.linkedin.com/feed/update/urn:li:activity:6745582134473445376/) - a great journey so far!

- Contributed well on GitHub: [700+ Commits, 70+ PRs & 50+ Issues](https://githubwrapped.tech/vinitshahdeo)!

- **Got [trending #1](https://twitter.com/Vinit_Shahdeo/status/1318493417412857858) developer on GitHub** - it was in my bucket list since the day I joined [GitHub](https://github.com/vinitshahdeo/).

- Received 26K+ [profile views](https://github.com/vinitshahdeo) on GitHub.

- Spoke at India's biggest [FOSS Hack](https://www.youtube.com/watch?v=wYEn2ASpSjM). Thanks @rhnvrm for this opportunity.

- Got featured in [newspaper](https://github.com/vinitshahdeo/COVID19#featured-on-newspaper) for building [COVID-19](https://github.com/vinitshahdeo/COVID19) tracker during very first week of Phase 1 of Lockdown 😷 

[![COVID-19 Tracker](https://github-readme-stats.vercel.app/api/pin/?username=vinitshahdeo&repo=COVID19)](https://github.com/vinitshahdeo/COVID19)

- Met some amazing developers - @ameerthehacker, @imabp, @praveenscience, @divyabhushan, @nainikaB, @smaranjitghose & @rajat2502! Hope to see you in person soon! 🤗 

- Mentored a winning team in [Smart India Hackathon 2020](https://twitter.com/Vinit_Shahdeo/status/1290860077709234178). Kudos to the Team Hashtag - @PragatiVerma18, @rajat2502, @shristisingh29, @MAJOR-BEAST, @abhinav223 & @Ryuk07 🍻 

[![AWSAR](https://github-readme-stats.vercel.app/api/pin/?username=rajat2502&repo=awsar)](https://github.com/rajat2502/awsar)

- Got featured in [Open Source For You](https://twitter.com/Vinit_Shahdeo/status/1289478475380744192) magazine - **[If you're a a techie, your home page should be GitHub not Instagram!](https://www.opensourceforu.com/2020/07/if-you-are-a-techie-your-home-page-should-be-github-not-instagram/)**

- Received 500+ PRs during [Hacktoberfest'20](https://dev.to/vinitshahdeo/hacktoberfest-2020-is-here-17op) :raised_hands:

[![Hacktoberfest 2020](https://github-readme-stats.vercel.app/api/pin/?username=vinitshahdeo&repo=Hacktoberfest2020)](https://github.com/vinitshahdeo/Hacktoberfest2020)

- Judged many hackathons including [Hack OnLoad 2.0](https://twitter.com/Vinit_Shahdeo/status/1237962227258286081), [Hack The Mountains](https://twitter.com/Vinit_Shahdeo/status/1315218679299072000) and [DSC WoW](https://twitter.com/Vinit_Shahdeo/status/1336170580387581952).

- First ever [Instagram Live](https://youtu.be/0VDtIJCsdTE) with @vinnovateit - [All about Placements with Vinit Shahdeo — VIT, Vellore](https://medium.com/@vinnovateit/all-about-placements-with-vinit-shahdeo-vit-vellore-87d3815d2436).

- Got featured in blogs - [Eat My News](https://www.eatmy.news/2020/06/code-like-you-eat-i-mean-code-daily-as.html), [TechnoVans](https://technovans.com/vinit-shahdeo-sharing-his-it-career-journey/) and [Launch2rise](https://launch2rise.in/2020/12/08/interview-with-vinit-shahdeo/).

- Published my first-ever article on Dev.to - [Code to generate the Map Of India using Node.js!](https://dev.to/vinitshahdeo/code-to-generate-the-map-of-india-using-node-js-3i06)

[![Map Of India](https://github-readme-stats.vercel.app/api/pin/?username=vinitshahdeo&repo=Map-of-India)](https://github.com/vinitshahdeo/Map-of-India)

- Trekked 6KM in 3 hours (all because of @SharmishthaDash) - it was fun! 😃 

- Conducted a contest([#MentorshipKarona](https://www.hackerrank.com/contests/mentorshipkarona-webinar-by-vinit-shahdeo/challenges)) on HackerRank. Congrats again to the top coders - @lakshyajit165 and @sukritishah15.

- Gained socially - 1K followers on [Twitter](https://twitter.com/Vinit_Shahdeo), 10K connections on [LinkedIn](https://www.linkedin.com/in/vinitshahdeo/) and ~1K followers on [GitHub](https://github.com/vinitshahdeo) (missed the goal 😞 ). Forever grateful to all who followed me. ❤️ 

- Maintained a [project](https://github.com/vinitshahdeo/Water-Monitoring-System) for [GirlScript Summer Of Code](https://www.linkedin.com/posts/vinitshahdeo_vinitshahdeowater-monitoring-system-activity-6652587525435166720-2eTy). Received awesome contributions from @goyalmuskan, @Swarnimashukla, @shreoshi-roy, @anmolkaur18, @PragatiVerma18,  @shreyakapoor08, @Adyasha8105 and many more!

[![Water-Monitoring-System](https://github-readme-stats.vercel.app/api/pin/?username=vinitshahdeo&repo=Water-Monitoring-System)](https://github.com/vinitshahdeo/Water-Monitoring-System)

- Laughed my heart out for [this](https://twitter.com/Vinit_Shahdeo/status/1275431072415707136) incident. Thanks to @romasinha for [putting this](https://qr.ae/pNJ2X7) on Quora. 🤩 

- First ever podcast with DSC Tiet - [Raw Talent - Episode 5 | Ft. Vinit Shahdeo](https://anchor.fm/dsctiet/episodes/Raw-Talent---Episode-5--Ft--Vinit-Shahdeo---Software-Engineering---Open-Source-and-Postman-Labs-elmsvt/a-a1ptlh). Thanks @AroraShreshth for the opportunity.

- Joined [Scaler Academy](https://www.scaler.com/edge/#path-guider) as a mentor. Grateful to all of my mentees - @yashvi2001, @ankan10, @ptk6, @pittala-prashanth & @Tejas8765

- Celebrated well for the win of @shreyaanand. She got placed in Apple! 🏆 

- Got featured as [#HumansOfVIT](https://twitter.com/Vinit_Shahdeo/status/1280350830764277760) by @vitspot. Thanks to @kashish121 for the push, not `git push`! 🤣 

- First ever [YouTube Live](https://www.youtube.com/watch?v=zsRv8g3C4rY) :red_circle: at [HackOn](https://twitter.com/Vinit_Shahdeo/status/1251819929247993856) hackathon. I was so nervous but it went well.

[![Event-Loop-In-JavaScript](https://github-readme-stats.vercel.app/api/pin/?username=vinitshahdeo&repo=Event-Loop-In-JavaScript)](https://github.com/vinitshahdeo/Event-Loop-In-JavaScript)

- Helped @imabp and @Souvikns to launch an open source organization, [Tech Phantoms](https://github.com/Tech-Phantoms).

- Delivered multiple webinars(by [StuCred](https://twitter.com/Vinit_Shahdeo/status/1258715954256547840), [Skillenza](https://www.hackerrank.com/mentorshipkarona-webinar-by-vinit-shahdeo/), etc.) on open-source and competitive coding. Thanks @prakarshpathak & @pvsmounish for organizing the sessions.

- **Met my childhood friends after 8 years**. Revisited the best moments of school days which made me laugh like insane after years.

- Mentored(1:1) 10+ folks including @chanchal1603, @hemshree, @imabp, @kashish121, @goyalmuskan, @SabhyaGrover, @Puja7629, @grey-hat2003 etc.

- Played [Skribbl](https://skribbl.io/) and [Among Us](https://play.google.com/store/apps/details?id=com.innersloth.spacemafia&hl=en_IN&gl=US) a lot. Unfortunately I lost every time I skribbled with @ShreyaAnand. 😿 

- Converted a college assignment into an open-source [project](https://github.com/vinitshahdeo/PortScanner). Thanks to @kashish121 and @ishika1727 for their contributions.

[![PortScanner](https://github-readme-stats.vercel.app/api/pin/?username=vinitshahdeo&repo=PortScanner)](https://github.com/vinitshahdeo/PortScanner)

- Received a [recommendation](https://www.linkedin.com/in/vinitshahdeo/) from @shristisingh29 - the best one so far! Tysm [Shristi](https://www.linkedin.com/in/shristi-singh-762987173/):)

- **Witnessed my elder sister in bridal lehanga**, a much-awaited breathtaking moment! ❤️ 

> I still remember the moment when DJ played [Din Shagna Da](https://youtu.be/X0MDALpV29s) and all the guests shifted their gaze to witness the first glimpse of bride all-set to put her first foot forward to the wedding venue! GOOSEBUMPS😭 

- Issued the first-ever cheque to my dad.

- Notified for a social media [mention](https://twitter.com/Vinit_Shahdeo/status/1342779388308254721) - a [caricature](https://www.instagram.com/p/CJPzvMllzS8/) of me!

- Survived 2020!

Before I start chapter 0 of XI, I want a moment to thank my [GitHub Fam](https://github.com/vinitshahdeo?tab=followers)! Wishing you all loads of happiness! May the new year be new start for all of us, all of you!

**Those who're tagged here, played vital role in making 2020 such a beautiful year. Blessed to have each one of you! Hope to see you in person in 2021!** ❤️ 

```js
 _   _                           _   _                __   __              _ 
| | | |                         | \ | |               \ \ / /             | |
| |_| | __ _ _ __  _ __  _   _  |  \| | _____      __  \ V /___  __ _ _ __| |
|  _  |/ _` | '_ \| '_ \| | | | | . ` |/ _ \ \ /\ / /   \ // _ \/ _` | '__| |
| | | | (_| | |_) | |_) | |_| | | |\  |  __/\ V  V /    | |  __/ (_| | |  |_|
\_| |_/\__,_| .__/| .__/ \__, | \_| \_/\___| \_/\_/     \_/\___|\__,_|_|  (_)
            | |   | |     __/ |                                              
            |_|   |_|    |___/                                               
```

Cheers to upcoming 2021:)🍻  This year, hit your goal!

#### **Before we meet and greet, would love to know your accomplishments of 2020?** Share what you've built, what will you be working in 2021?

❤️ 💙 💛 💚 

_Forever grateful,_
_**Vinit Shahdeo**_

**Comment by smaranjitghose** [comment]
> # 👋 Hello!
> 2020 : Thanks for the experiences, good or bad!
> **2021 : Welcome, here I go!**
> 
> This year wasn't like any year before, but we learnt a lot about how we can work together in new ways to make things work. This time last year, concepts such as _"lockdown", "mask mandates"_ and _"social distancing"_ were unknown to most of us. Today they are part of our everyday language as the COVID-19 pandemic continues to impact all aspects of our lives.
> 
> 2020, what a year this was! Despite all the problems we faced, **I got a chance to spend time with my family after 8 years.** ...and my elder sister [got married](https://github.com/vinitshahdeo/Wedding-Invitation) this year! I'm grateful for everything:) ❤️
> 
> ### Here's a recap of my 2020:
> * Completed [2 years at Postman](https://www.linkedin.com/feed/update/urn:li:activity:6745582134473445376/) - a great journey so far!
> * Contributed well on GitHub: [700+ Commits, 70+ PRs & 50+ Issues](https://githubwrapped.tech/vinitshahdeo)!
> * **Got [trending #1](https://twitter.com/Vinit_Shahdeo/status/1318493417412857858) developer on GitHub** - it was in my bucket list since the day I joined [GitHub](https://github.com/vinitshahdeo/).
> * Received 26K+ [profile views](https://github.com/vinitshahdeo) on GitHub.
> * Spoke at India's biggest [FOSS Hack](https://www.youtube.com/watch?v=wYEn2ASpSjM). Thanks @rhnvrm for this opportunity.
> * Got featured in [newspaper](https://github.com/vinitshahdeo/COVID19#featured-on-newspaper) for building [COVID-19](https://github.com/vinitshahdeo/COVID19) tracker during very first week of Phase 1 of Lockdown 😷
> 
> [![COVID-19 Tracker](https://camo.githubusercontent.com/361fafb3f531534034fe34c81ed3133112e856761ed56f22e1fc3ad4212ce946/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170692f70696e2f3f757365726e616d653d76696e69747368616864656f267265706f3d434f5649443139)](https://github.com/vinitshahdeo/COVID19)
> 
> * Met some amazing developers - @ameerthehacker, @imabp, @praveenscience, @divyabhushan, @nainikaB, @smaranjitghose & @rajat2502! Hope to see you in person soon! 🤗
> * Mentored a winning team in [Smart India Hackathon 2020](https://twitter.com/Vinit_Shahdeo/status/1290860077709234178). Kudos to the Team Hashtag - @PragatiVerma18, @rajat2502, @shristisingh29, @MAJOR-BEAST, @abhinav223 & @Ryuk07 🍻
> 
> [![AWSAR](https://camo.githubusercontent.com/90a717625c023937114e48d16986ea05ed856043d1c41cecd8c3fe777b03dd3b/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170692f70696e2f3f757365726e616d653d72616a617432353032267265706f3d6177736172)](https://github.com/rajat2502/awsar)
> 
> * Got featured in [Open Source For You](https://twitter.com/Vinit_Shahdeo/status/1289478475380744192) magazine - **[If you're a a techie, your home page should be GitHub not Instagram!](https://www.opensourceforu.com/2020/07/if-you-are-a-techie-your-home-page-should-be-github-not-instagram/)**
> * Received 500+ PRs during [Hacktoberfest'20](https://dev.to/vinitshahdeo/hacktoberfest-2020-is-here-17op) 🙌
> 
> [![Hacktoberfest 2020](https://camo.githubusercontent.com/a29d30543533a07ecdbc274968424647a3a36d6bcabe6b3791db53db30d6404d/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170692f70696e2f3f757365726e616d653d76696e69747368616864656f267265706f3d4861636b746f6265726665737432303230)](https://github.com/vinitshahdeo/Hacktoberfest2020)
> 
> * Judged many hackathons including [Hack OnLoad 2.0](https://twitter.com/Vinit_Shahdeo/status/1237962227258286081), [Hack The Mountains](https://twitter.com/Vinit_Shahdeo/status/1315218679299072000) and [DSC WoW](https://twitter.com/Vinit_Shahdeo/status/1336170580387581952).
> * First ever [Instagram Live](https://youtu.be/0VDtIJCsdTE) with @vinnovateit - [All about Placements with Vinit Shahdeo — VIT, Vellore](https://medium.com/@vinnovateit/all-about-placements-with-vinit-shahdeo-vit-vellore-87d3815d2436).
> * Got featured in blogs - [Eat My News](https://www.eatmy.news/2020/06/code-like-you-eat-i-mean-code-daily-as.html), [TechnoVans](https://technovans.com/vinit-shahdeo-sharing-his-it-career-journey/) and [Launch2rise](https://launch2rise.in/2020/12/08/interview-with-vinit-shahdeo/).
> * Published my first-ever article on Dev.to - [Code to generate the Map Of India using Node.js!](https://dev.to/vinitshahdeo/code-to-generate-the-map-of-india-using-node-js-3i06)
> 
> [![Map Of India](https://camo.githubusercontent.com/5b0df3f0c790c66d14a601e34d3b8d5ace19f8d556b6c991e29242afdf302da5/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170692f70696e2f3f757365726e616d653d76696e69747368616864656f267265706f3d4d61702d6f662d496e646961)](https://github.com/vinitshahdeo/Map-of-India)
> 
> * Trekked 6KM in 3 hours (all because of @sharmishthadash) - it was fun! 😃
> * Conducted a contest([#MentorshipKarona](https://www.hackerrank.com/contests/mentorshipkarona-webinar-by-vinit-shahdeo/challenges)) on HackerRank. Congrats again to the top coders - @lakshyajit165 and @sukritishah15.
> * Gained socially - 1K followers on [Twitter](https://twitter.com/Vinit_Shahdeo), 10K connections on [LinkedIn](https://www.linkedin.com/in/vinitshahdeo/) and ~1K followers on [GitHub](https://github.com/vinitshahdeo) (missed the goal 😞 ). Forever grateful to all who followed me. ❤️
> * Maintained a [project](https://github.com/vinitshahdeo/Water-Monitoring-System) for [GirlScript Summer Of Code](https://www.linkedin.com/posts/vinitshahdeo_vinitshahdeowater-monitoring-system-activity-6652587525435166720-2eTy). Received awesome contributions from @goyalmuskan, @Swarnimashukla, @shreoshi-roy, @anmolkaur18, @PragatiVerma18,  @shreyakapoor08, @Adyasha8105 and many more!
> 
> [![Water-Monitoring-System](https://camo.githubusercontent.com/638dee34ac3cd6d2a2d7b1dc2d668cc6412f1b6b93e4ebf7ab27f87f2439afa7/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170692f70696e2f3f757365726e616d653d76696e69747368616864656f267265706f3d57617465722d4d6f6e69746f72696e672d53797374656d)](https://github.com/vinitshahdeo/Water-Monitoring-System)
> 
> * Laughed my heart out for [this](https://twitter.com/Vinit_Shahdeo/status/1275431072415707136) incident. Thanks to @romasinha for [putting this](https://qr.ae/pNJ2X7) on Quora. 🤩
> * First ever podcast with DSC Tiet - [Raw Talent - Episode 5 | Ft. Vinit Shahdeo](https://anchor.fm/dsctiet/episodes/Raw-Talent---Episode-5--Ft--Vinit-Shahdeo---Software-Engineering---Open-Source-and-Postman-Labs-elmsvt/a-a1ptlh). Thanks @AroraShreshth for the opportunity.
> * Joined [Scaler Academy](https://www.scaler.com/edge/#path-guider) as a mentor. Grateful to all of my mentees - @yashvi2001, @ankan10, @ptk6, @pittala-prashanth & @Tejas8765
> * Celebrated well for the win of @ShreyaAnand. She got placed in Apple! 🏆
> * Got featured as [#HumansOfVIT](https://twitter.com/Vinit_Shahdeo/status/1280350830764277760) by @vitspot. Thanks to @Kashish121 for the push, not `git push`! 🤣
> * First ever [YouTube Live](https://www.youtube.com/watch?v=zsRv8g3C4rY) 🔴 at [HackOn](https://twitter.com/Vinit_Shahdeo/status/1251819929247993856) hackathon. I was so nervous but it went well.
> 
> [![Event-Loop-In-JavaScript](https://camo.githubusercontent.com/2c778462176d9e4874dd26e681681cdb6a5a1fceaa47250b7664a621a4be2598/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170692f70696e2f3f757365726e616d653d76696e69747368616864656f267265706f3d4576656e742d4c6f6f702d496e2d4a617661536372697074)](https://github.com/vinitshahdeo/Event-Loop-In-JavaScript)
> 
> * Helped @imabp and @Souvikns to launch an open source organization, [Tech Phantoms](https://github.com/Tech-Phantoms).
> * Delivered multiple webinars(by [StuCred](https://twitter.com/Vinit_Shahdeo/status/1258715954256547840), [Skillenza](https://www.hackerrank.com/mentorshipkarona-webinar-by-vinit-shahdeo/), etc.) on open-source and competitive coding. Thanks @prakarshpathak & @pvsmounish for organizing the sessions.
> * **Met my childhood friends after 8 years**. Revisited the best moments of school days which made me laugh like insane after years.
> * Mentored(1:1) 10+ folks including @Chanchal1603, @Hemshree, @imabp, @Kashish121, @goyalmuskan, @SabhyaGrover, @Puja7629, @grey-hat2003 etc.
> * Played [Skribbl](https://skribbl.io/) and [Among Us](https://play.google.com/store/apps/details?id=com.innersloth.spacemafia&hl=en_IN&gl=US) a lot. Unfortunately I lost every time I skribbled with @ShreyaAnand. 😿
> * Converted a college assignment into an open-source [project](https://github.com/vinitshahdeo/PortScanner). Thanks to @Kashish121 and @ishika1727 for their contributions.
> 
> [![PortScanner](https://camo.githubusercontent.com/b8812c84d6577768d412ceb241cdffe90de589ea9ca3ff5fa7d1e1de31196fcf/68747470733a2f2f6769746875622d726561646d652d73746174732e76657263656c2e6170702f6170692f70696e2f3f757365726e616d653d76696e69747368616864656f267265706f3d506f72745363616e6e6572)](https://github.com/vinitshahdeo/PortScanner)
> 
> * Received a [recommendation](https://www.linkedin.com/in/vinitshahdeo/) from @shristisingh29 - the best one so far! Tysm [Shristi](https://www.linkedin.com/in/shristi-singh-762987173/):)
> * **Witnessed my elder sister in bridal lehanga**, a much-awaited breathtaking moment! ❤️
> 
> > I still remember the moment when DJ played [Din Shagna Da](https://youtu.be/X0MDALpV29s) and all the guests shifted their gaze to witness the first glimpse of bride all-set to put her first foot forward to the wedding venue! GOOSEBUMPS😭
> 
> * Issued the first-ever cheque to my dad.
> * Notified for a social media [mention](https://twitter.com/Vinit_Shahdeo/status/1342779388308254721) - a [caricature](https://www.instagram.com/p/CJPzvMllzS8/) of me!
> * Survived 2020!
> 
> Before I start chapter 0 of XI, I want a moment to thank my [GitHub Fam](https://github.com/vinitshahdeo?tab=followers)! Wishing you all loads of happiness! May the new year be new start for all of us, all of you!
> 
> **Those who're tagged here, played vital role in making 2020 such a beautiful year. Blessed to have each one of you! Hope to see you in person in 2021!** ❤️
> 
> ```js
>  _   _                           _   _                __   __              _ 
> | | | |                         | \ | |               \ \ / /             | |
> | |_| | __ _ _ __  _ __  _   _  |  \| | _____      __  \ V /___  __ _ _ __| |
> |  _  |/ _` | '_ \| '_ \| | | | | . ` |/ _ \ \ /\ / /   \ // _ \/ _` | '__| |
> | | | | (_| | |_) | |_) | |_| | | |\  |  __/\ V  V /    | |  __/ (_| | |  |_|
> \_| |_/\__,_| .__/| .__/ \__, | \_| \_/\___| \_/\_/     \_/\___|\__,_|_|  (_)
>             | |   | |     __/ |                                              
>             |_|   |_|    |___/                                               
> ```
> 
> Cheers to upcoming 2021:)🍻 This year, hit your goal!
> 
> #### **Before we meet and greet, would love to know your accomplishments of 2020?** Share what you've built, what will you be working in 2021?
> ❤️ 💙 💛 💚
> 
> _Forever grateful,_
> _**Vinit Shahdeo**_

It was a complete pleasure knowing such a down to earth and talented person like you @vinitshahdeo. 
Cheers to those VIT-SRM discussions🔥
Happy new year in advance brother! I wish you the best in health and opportunities.

**Comment by Puja7629** [comment]
Such a lovely, amazing, talented, very hard working and completely down to
earth person I have ever met. I wish you all the success and happiness in
the coming years. I am glad to this 2020 that I met you. I am looking
forward to learning a lot from you in the future.
Stay awesome, Stay amazing the way you are.
Thanks for existing in 2020 :p

HAPPY NEW YEAR IN ADVANCE, VINIT :)

**Comment by imabp** [comment]
Thank you @vinitshahdeo. 
Always it's a happy feeling to be with you 😊

You are a friend, mentor, guide, brother, and whatnot. Always ready to help me out in different situations.❤
Congratulations!! For dii's wedding. I really loved seeing those quick LIVE on Instagram.😋😋😁

From the launch of the organization [Tech Phantoms](https://github.com/tech-phantoms) till date, you have been guiding us and will always be. 😊😊
It's an honor to have you with us. 😊😊😊

Really it's on my bucket list, to meet you (@vinitshahdeo)  and @MadhavBahl in person.

Reading the line from your message
> Issued the first-ever cheque to my dad.

gave me goosebumps 😱😱. Congratulations 🎉🎉, its a big thing!!! 😁😁

It was a surprise to see the mention in the amazing developer's section. Still learning is on! 😁😁 
Hope to have more projects up with you and build super cool things using APIs.

@vinitshahdeo  @madhavbahl 
Heres a snippet, from my side to run on browser's console 😊😊

```js

function happynewyear( family ){
family.push("health");
family.push("wealth");
family.push("happiness");
console.log("🎉Wishing Happy New Year to  🎉  "+ family[0] + ". I wish sound "+family[1]+", "+family[2]+" and "+family[3]+ " for your family "  )
return true
}
const mentors = [ [ " Vinit Shahdeo " ]  , [ " Madhav Bahl " ] ]

mentors.map(mentor=>happynewyear(mentor))


```

**Reply by MadhavBahl** [reply]
Would love to meet you as well, @imabp ✨ (And, thanks for such a beautiful greeting, I am flattered 🙈) 

@vinitshahdeo , you are doing some real great work, really inspired 💯

**Comment by Hemshree** [comment]
It's absolutely a pleasure for me to meet such an amazing, super-talented and an absolutely down to earth personality. I'm thankful to COVID-19 because it gave me a reason to know you. Keep up the great work you're doing, @vinitshahdeo 😄 

Thanks for being a great mentor and a helpful friend. You're really an inspirational soul. Stay like that! 

I wish all the good things come to you this New Year. Wishing you and your loved ones a very happy new year! 😊

**Comment by SabhyaGrover** [comment]
Thanks a ton @vinitshahdeo . 
I am really thankful for all the guidance and valuable advices. You are an inspiration and I am very grateful to have known you this year. 
Wish you and your family a very Happy New Year !

**Comment by divyabhushan-okta** [comment]
Hey @vinitshahdeo, 
I am and will always remain thankful to you for trusting me and providing me the opportunity to be a part of your genius works. You are a true inspiration, mentor, genius, down-to-earth, talented, and a very humble person. It's my honor to be associated with you. I wish you, your family, and your team all the success, health, and prosperity ahead and always. I also look forward to meeting you in person and contribute and learn from you 👍

**Comment by sukritishah15** [comment]
I am extremely grateful to you @vinitshahdeo !
In you, I have found a mentor. I have very closely followed your GitHub profile and learnt a lot from you. 
Your energy is contagious!
Your work is inspirational !

#MentorshipKarona was a nice contest, consisting of a bunch of interesting questions (with even more interesting references and cute pictures of food), and it came in around a time, while I was preparing for my interviews and so, it helped me practice.

I genuinely look forward to learn more from you and contribute to your repositories in the near future.
Wish you a very Happy New Year!

**Comment by ameerthehacker** [comment]
Thank you buddy, the pleasure was mine I totally enjoyed our meeting at hackonload and very much lookinh forward to meeting you again. Have a great year ahead mate and looking forward to your new awesome projects to stalk 😸

**Comment by Chanchal1603** [comment]
Hey @vinitshahdeo ,
Thank you for everything, You are one of the good friends and best mentor that I ever got. You're a complete package of talent, genius mind, friendly nature, humble and a very helpful person. No doubt 2020 gave me opportunity to connect with many amazing bright minded people like you. Wishing you and your family a cheerful year ahead🤗
Hoping to meet you in person this year:)

**Comment by nainikaB** [comment]
Hey @vinitshahdeo 
Thank you so much for being such a great friend! So glad that 2020 got such an amazing person into my life! Your talent, drive and contribution to the community is what’s almost admirable! ❤️ Happy new year and hope you have a great year ahead! 🤗

Hoping to meet in you in person and have chai with you ☕️
